### PR TITLE
organize protobuf files

### PR DESCRIPTION
**Pull request by dhpollack**
Issue #297 

To make the protobufs use relative imports, I did the following.  If you
regenerate the protobuf files then you will have to run the last command
to get the relative imports to work correctly.

```sh
cd presidio-analyzer/presidio_analyzer
mkdir -p protobuf_models
mv *_pb2*.py protobuf_models/
cd protobuf_models
sed -i -E 's/^import.*_pb2/from . \0/' *.py
```

Signed-off-by: David Pollack <d.pollack@solvemate.com>

### CODE REVIEW

**Comment by eladiw** [comment]
thanks @dhpollack we will review this PR soon.
Thanks for your contribution!

File: presidio-analyzer/presidio_analyzer/protobuf_models/analyze_pb2_grpc.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 import grpc
 
-import analyze_pb2 as analyze__pb2
+from . import analyze_pb2 as analyze__pb2
```
Comment (eladiw) [inline presidio-analyzer/presidio_analyzer/protobuf_models/analyze_pb2_grpc.py:4]: was this regenerated?

Comment (eladiw) [reply presidio-analyzer/presidio_analyzer/protobuf_models/analyze_pb2_grpc.py:4]: just saw you used sed to alter these files

Comment (eladiw) [reply presidio-analyzer/presidio_analyzer/protobuf_models/analyze_pb2_grpc.py:4]: why is this a must. I'd prefer to have as little manual intervention as possible

Comment (dhpollack) [reply presidio-analyzer/presidio_analyzer/protobuf_models/analyze_pb2_grpc.py:4]: No, I used sed command that's in the comment

**Comment by dhpollack** [comment]
The manual step is not ideal but it's a long standing issue with Google and how they generate the files. You can see the link in the issue that's attached to this PR. Instead of adding this functionality they insist that it's not need despite the hundreds of messages in the issue.

**Comment by omri374** [comment]
@dhpollack If this fix: https://github.com/protocolbuffers/protobuf/issues/1491#issuecomment-542398041 would work, it's easier to maintain and doesn't require some post processing. WDYT?

**Comment by dhpollack** [comment]
> @dhpollack If this fix: [protocolbuffers/protobuf#1491 (comment)](https://github.com/protocolbuffers/protobuf/issues/1491#issuecomment-542398041) would work, it's easier to maintain and doesn't require some post processing. WDYT?

The comment after that one looks more fool-proof.  I'll give that a try.

**Comment by dhpollack** [comment]
It looks like the build is failing due to failure downloading the spacy models, even though all the tests are passing.  It would be nice if you used the small spacy model as a default, so one can test presidio without having to download and build 800MB large model.  Also, the English spacy models aren't actually "requirements" if someone were to use this library on non-English data.

**Comment by omri374** [comment]
We have an issue with the build agent which is currently being fixed (see https://github.com/microsoft/presidio/pull/300)

The reason we're using the big spaCy model is because it has NER capabilities which the small one doesn't, and some of our tests validate the output of the spaCy model. You are on spot regarding other languages. The current deployment is coupled with the English version and supporting deployments of other models requires some changes to the deployment pipeline.

**Comment by eladiw** [comment]
Hi @dhpollack,
We have fixed the build issue. can you fix the merge conflicts and we will merge this

**Comment by dhpollack** [comment]
@eladiw I rebased on master and then fixed the imports in the test.  I think it should work now.

**Pull request merged**